modules/exploits/multi/misc/w3tw0rk_exec.rb

```diff
@@ -0,0 +1,160 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = AverageRanking
```

Comment:
Ohw I see, I should change this to ExcellentRanking then because it is a CMD execution. Thanks for the heads up @wchen-r7